demos/sev-demo/README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+Please visit this page for the most up to date instructions: https://github.com/confidential-containers/documentation/blob/main/quickstart.md#sev
```

Comment:
Can we add something about why we are leaving this document and not just deleting it, like it's an overview of the changes that are made for SEV or something like that.

Reply:
I'd be in favour of actually removing the content, as it may end up confusing whoever is trying to use it.

Reply:
Yeah I am fine with deleting.